session/manager_test.go

```diff
@@ -30,11 +30,12 @@ var (
 	currentProposal   = market.ServiceProposal{
 		ID: currentProposalID,
 	}
+	consumerID = identity.FromAddress("deadbeef")
+
 	expectedID      = ID("mocked-id")
 	expectedSession = Session{
 		ID:         expectedID,
-		Config:     expectedSessionConfig,
-		ConsumerID: identity.FromAddress("deadbeef"),
+		ConsumerID: consumerID,
 	}
 	lastSession Session
```

Comment:
This `lastSession` is unused can we remove it in this PR? 
`const expectedSessionConfig = "config_string"` is unused too.